src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.Paint.cs

```diff
@@ -24,6 +24,18 @@ public partial class ShadowContainer
 	private ShadowPaintState? _lastPaintState;
 	private bool _isShadowDirty;
 
+	internal event EventHandler<SurfacePaintCompletedEventArgs>? SurfacePaintCompleted;
```

Comment:
Do we want to make a custom handler delegate for this? Like [here](https://github.com/unoplatform/uno.toolkit.ui/blob/2c9c84a0efd057d8f3cbf59db6a4f44f448d5a33/src/Uno.Toolkit.UI/Controls/Chips/Chips.EventHandlers.cs#L8)? Or does it not matter?

Reply:
I didn't understand the point or purpose.

Reply:
I thought we wanted to avoid generics like this, but I might be getting confused with something else. @jeromelaban is there anything to do here or it's fine as is?